api/openapi/solidus-api.oas.yml

```diff
@@ -5579,6 +5579,10 @@ paths:
         Creates a shipment.
 
         Please note that this request can be only performed by users with the `create` permission on the shipment.
+
+        **Deprecation Warning**: Adding items to the shipment via this endpoint
+        is deprecated. Instead, create an empty shipment and populate it with
+        the dedicated endpoint [to add items to the shipment](/docs/solidus/7078dbcf415ac-add-shipment-item).
```

Comment:
I don't know if this way of adding link will work. I saw [the docs here](https://meta.stoplight.io/docs/platform/hosted-docs/stoplight-urls) but I don't know how to test it until it's published.

Reply:
Oh, interesting. I was looking for something like it for #4385 & #4388. If it works, I'll update the references I added there.

Reply:
What do you mean? The deprecation warning is only rendered on `master`, and I can switch the version with the top-left selector without problems.

I submitted https://github.com/solidusio/solidus/pull/4397 to update my references.

Reply:
I mean that if you click on the link from the `master` version, it will move you to the default one (currently 3.1) because the `/branches/master/` part of the URL is not reflected in the link.

Reply:
Oh, I see :facepalm:

Reply:
Yeah, I thought exactly that :wink: